packages/scss/src/components/indexTableSorted/component.scss

```diff
@@ -0,0 +1,48 @@
+@use '@lucca-front/scss/src/components/button/exports' as button;
+@use '@lucca-front/icons/src/icon/exports' as icon;
+
+@mixin component($atRoot: 'without: rule') {
+    @include button.text;
+    @include button.withIcon;
+
+    --components-button-padding: 0 var(--pr-t-spacings-50);
+    --components-button-gap: var(--pr-t-spacings-25);
+
+    user-select: none;
+    font-weight: 400;
+    vertical-align: top;
+    align-items: flex-start;
+    left: var(--components-sorted-offset);
+    text-align: inherit;
+    flex-direction: var(--components-sorted-direction);
```

Comment:
Indentation is too large here (5-17).

Reply:
Actually, this file hesitate between 2 spaces and 4 spaces indentation.

(Maybe adding some tabs would be fun! :p)